ansible/roles/whiskhost/tasks/deploy.yml

```diff
@@ -2,6 +2,6 @@
 # Add user and patch kernel.
 
 - include: addUser.yml
-  when: "'prod' in deployment"
+  when: docker.user is defined
```

Comment:
@lehoanganh is right about that. it should be similar to the docker_image_tag param except it does not have a default value. I would also prefer to set it on the environment even though for us the user is probably going to be the same everywhere.